### DISCUSSION TITLE
RTSP Setup

### DISCUSSION THREAD

**Discussion by johnmac41** [mcguirepr89/BirdNET-Pi#1100] [Q&A]
I have  a Amcrest IP Camera with microphone and would like to try it out on BirdNet. I searched for installation instructions with no luck. I see where to put the stream URL in settings, but have had no luck. I have tried a couple of scenarios of urls that did not work. Amcrest is not too forthcoming with info on RTSP that I could find. here is one url I tried - "rtsp://username:password@192.168.1.34/cam/realmonitor?channel=1&subtype=0&unicast=true&proto=Onvif"

Looking for any help and/or sources to set this up.

Thanks, Johnmac78

**Comment by pddpauw** [comment]
suggestion: try e.g. with vlc if you could listen live to this stream.  If not (or if you get any error), then you know the rtsp address is not right.

**Reply by johnmac41** [reply]
It works with VLC. Not sure what I need for audio. I have it set for default for pulseaudio, but have no idea what I need to do to config.
The "live audio" button has disappeared?

**Reply by pddpauw** [reply]
based on the reply, I presume you mean with "It works with VLC. Not sure what I need for audio." that you have visual and audio in VLC, correct?  In this case, you would need to try to find the audio channel of your camera...  I'm sure google and some try and error in VLC can  help.

**Comment by simongis** [comment]
Was wondering if you found a resolution to this and if you could share your setup/URL you used @johnmac41?

**Comment by johnmac41** [comment]
No, I did not get it to work. May try again this spring.
